client/packages/android/package.json

```diff
@@ -9,7 +9,9 @@
     "@capacitor/app": "^6.0.0",
     "@capacitor/cli": "^6.1.0",
     "@capacitor/keyboard": "^6.0.1",
-    "@capacitor/preferences": "^6.0.1"
+    "@capacitor/preferences": "^6.0.1",
+    "@capacitor-community/file-opener": "6",
+    "@capacitor/filesystem": "6"
```

Comment:
Do we need explicit version numbers here, you think? (e.g. "6.0.3")

Reply:
I'm not entirely sure if we need to specify the exact version. Personally I just followed the versioning pattern used, allowing it to increment around this version.